recipes-openxt/openxt/openxt-keymanagement/key-functions

```diff
@@ -17,6 +17,24 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
+
+is_tpm_2_0 () {
+   [ -e /sys/class/tpm/tpm0/device/description ] && cat /sys/class/tpm/tpm0/device/description | grep "2.0" &>/dev/null
+}
+
+pcr_bank_exists () {
+    local alg_in=$1
+
+    banks=$(tpm2_listpcrs -s | cut -d ':' -f 2)
+    for bank in $banks; do
+        alg=$(echo $bank | cut -d '(' -f 1)
+        if [ "$alg" = $alg_in ]; then
+            return 0
+        fi
+    done
+    return 1
+}
+
```

Comment:
@crogers1 and @rossphilipson , if the attempt doesn't work, then yes we can punt and put in a major ticket to fix as it doesn't break anything except the design concept of modular isolation.